cirq-core/cirq/sim/__init__.py

```diff
@@ -68,6 +68,8 @@
 
 from cirq.sim.state_vector_simulation_state import StateVectorSimulationState
 
+from cirq.sim.classical_simulator import ClassicalSimulator, _run
```

Comment:
Remove _run from this line.  Anything with an underscore is generally package-private and should not be re-exported.  Also, _run is a member function of ClassicalSimulator and not a global function.